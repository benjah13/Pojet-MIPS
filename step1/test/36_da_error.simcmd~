# test da erreur arg1 

da ae1:20
da ae1:20
