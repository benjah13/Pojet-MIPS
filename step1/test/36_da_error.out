DA function should display 20 instructions from address 0xae1
Invalid param : hexadecimal number awaited in 'ae1v'
